git_machete/cmd.py

```diff
@@ -2281,8 +2281,9 @@ def strip_first_fragment(rb: str) -> str:
             cb = current_branch(git)
             rebase_onto_ancestor_commit(git, cb, cli_opts.opt_fork_point or machete_client.fork_point(cb, use_overrides=True))
         elif cmd == "show":
-            param = check_required_param(args[:1], allowed_directions(allow_current=True))
-            branch = check_optional_param(args[1:])
+            args1 = parse_options(args)
```

Comment:
Whoops

Reply:
Hmm, is it necessary to call parse_options again? All in all it's called before   at 2122
`cmd_and_args = parse_options(orig_args, gnu=False)`

Reply:
It is necessary... let me explain in the code why; anyway, I hope this will all be gone with the advent of #158